kumascript/macros/CSSRef.ejs

```diff
@@ -1,283 +1,582 @@
-                if (group.indexOf(typeGroup[g]) !== -1 &&
-                        groupData.indexOf(type) === -1) {
-                    groupData.push(type);
-                }
-            }
-        }
-    }
+function buildSublist(pages, title) {
+  let result = `<li class="toggle"><details><summary>${title}</summary><ol>`;
 
-    // Load the CSS pages data, gather info for the found properties and atRules
-    var pageList = await page.subpagesExpand('/en-US/docs/Web/CSS');
-    for (var i in pageList) {
-      var aPage = pageList[i];
-      for (var x in groupData) {
-        if (aPage.slug.split("/").pop() === groupData[x].replace(/ /g, '_')) {
-          if (isAtRule(groupData[x])) {
-            atRules.push(aPage);
-          } else if (aPage.slug.indexOf('selectors') !== -1 ||
-                aPage.slug.indexOf('combinator') !== -1 ||
-                aPage.slug.indexOf(':') !== -1 ) {
-            selectors.push(aPage);
-          } else {
-            properties.push(aPage);
-          }
-        } else if (aPage.title === groupData[x] && isType(groupData[x])) {
-            types.push(aPage);
-        }
-      }
-    }
+  for (const page of pages) {
+    const url = page.url.replace('en-US', locale);
+    const title = htmlEscape(page.title);
 
-    if (isSelector) {
-      var selectorData = data.selectors;
-      var selectorCollection = {};
-      for (var selector in selectorData) {
-          var g = selectorData[selector].groups[0];
-          for (s in selectors) {
-            var aPage = selectors[s];
-            if(aPage.slug.split("/").pop().replace(/_/g, ' ') === selector) {
-                if (g in selectorCollection) {
-                    selectorCollection[g].push(aPage);
-                } else {
-                    selectorCollection[g] = [];
-                    selectorCollection[g].push(aPage);
-                }
+    const translated = false;
+    if (locale != 'en-US') {
+        page.translations.forEach(function(translation){
+            if (translation.locale === locale) {
+                url = translation.url;
+                title = htmlEscape(translation.title);
+                translated = true;
             }
-          }
-      }
-    }
-
-    // Load sub pages of the group
-    var groupPages = [];
-    for (var i = 0; i < group.length; i++) {
-        groupPages = groupPages.concat(await page.subpagesExpand('/en-US/docs/Web/CSS/' + group[i].replace(/ /g, '_')));
+        });
     }
 
-    // output helpers
-    var badges = {
-      ExperimentalBadge : await template("ExperimentalBadge"),
-      NonStandardBadge : await template("NonStandardBadge"),
-      DeprecatedBadge : await template("DeprecatedBadge"),
-      ObsoleteBadge : await template("ObsoleteBadge"),
+    const cta = '';
+     if (!translated && locale != 'en-US') {
+        cta += ' <a href="'+ url +'$translate" style="opacity:0.5" title="'+ text['title'] + '">' + text['translate'] + '</a>';
```

Comment:
drive-by comment; any mention of `$translate` can be removed.

Reply:
Maybe we can remove all this `cta` stuff? IDK I just cargo-culted it.